ci/scale-pure-ovn-hosts.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Save trace setting
+XTRACE=$(set +o | grep xtrace)
+set -o xtrace
+
+# Read variables
+source ovn-scale.conf
+
+# Library files
+source scale-lib.sh
```

Comment:
@mestery sorry, I put the comment below the wrong line. Since the functions (e.g., ```check_container_failure```) are moved to scale-lib.sh, we should source the file in original driver file, which is docker-rally.sh